### PR TITLE
Split Az.DataBoxEdge Examples

### PR DESCRIPTION
**Pull request by CaptainFanZzz**
## Description

Split Az.DataBoxEdge Examples

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeShare.md [hunk 1]
```diff
@@ -48,9 +48,12 @@ The **New-AzDataBoxEdgeShare** cmdlet creates a new share on the Data Box Edge d
 ## EXAMPLES
 
 ### Example 1
-```
-PS C:\> New-AzDataBoxEdgeShare -ResourceGroupName resourceGroupName -DeviceName deviceName -Name share-1 -SMB
+```powershell
+New-AzDataBoxEdgeShare -ResourceGroupName resourceGroupName -DeviceName deviceName -Name share-1 -SMB
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeShare.md:52]: add " `" to the end

File: src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeUser.md [hunk 2]
```diff
@@ -25,16 +25,22 @@ The **New-AzDataBoxEdgeUser** cmdlet creates a new user for the Data Box Edge de
 
 ### Example 1
 ```powershell
-PS C:\> New-AzDataBoxEdgeUser -ResourceGroupName resourceGroupName -DeviceName deviceName -Name username
+New-AzDataBoxEdgeUser -ResourceGroupName resourceGroupName -DeviceName deviceName -Name username
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeUser.md:28]: add " `" to the end

File: src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeUser.md [hunk 3]
```diff
@@ -25,16 +25,22 @@ The **New-AzDataBoxEdgeUser** cmdlet creates a new user for the Data Box Edge de
...
+
+```output
 User name   Type  ResourceGroupName DeviceName
 ---------   ----  ----------------- ----------
 username    Share resourceGroupName deviceName
 ```
 
 ```powershell
-PS C:\> New-AzDataBoxEdgeUser -ResourceGroupName resourceGroupName -DeviceName deviceName -Name username
+New-AzDataBoxEdgeUser -ResourceGroupName resourceGroupName -DeviceName deviceName -Name username
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/New-AzDataBoxEdgeUser.md:39]: add " `" to the end

File: src/DataBoxEdge/DataBoxEdge/help/Set-AzDataBoxEdgeStorageAccountCredential.md [hunk 4]
```diff
@@ -40,8 +40,11 @@ The **Set-AzDataBoxEdgeStorageAccountCredential** cmdlet updates the storage acc
 
 ### Example 1
 ```powershell
-PS C:\> Set-AzDataBoxEdgeStorageAccountCredential -ResourceGroupName resourceGroupName -DeviceName deviceName -Name storageAcountCredentialName
+Set-AzDataBoxEdgeStorageAccountCredential -ResourceGroupName resourceGroupName -DeviceName deviceName -Name storageAcountCredentialName
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/Set-AzDataBoxEdgeStorageAccountCredential.md:43]: add " `" to the end

File: src/DataBoxEdge/DataBoxEdge/help/Set-AzDataBoxEdgeUser.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ The **Set-AzDataBoxEdgeUser** cmdlet sets a new password for a user on the Data
 
 ### Example 1
 ```powershell
-PS C:\> Set-AzDataBoxEdgeUser -ResourceGroupName resourceGroupName -DeviceName deviceName -Name username
+Set-AzDataBoxEdgeUser -ResourceGroupName resourceGroupName -DeviceName deviceName -Name username
```
Comment (ziyuezh576) [inline src/DataBoxEdge/DataBoxEdge/help/Set-AzDataBoxEdgeUser.md:41]: add " `" to the end

**Review by ziyuezh576 — CHANGES_REQUESTED** [review]

**Comment by CaptainFanZzz** [comment]
@ziyuezh576 Changed. Plz check.

**Review by ziyuezh576 — APPROVED** [review]

**Pull request merged**